src/components/02-components/card-experiences-list/card-experiences-list.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Paper } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import CardExperiences from '../card-expreciences/card-experiences';
+
+
+export type BrandProps = {
+  cardList: Array<object>,
```

Comment:
Be more specific. What kind of object? 

You can import the type from the experiences card component, to avoid the need to define the whole object again here:

```
import { type  CardExperienceProps } from '../card-experience/card-experience';

...

cardList: CardExperienceProps[];
```